### DISCUSSION TITLE
ssh key pair

### DISCUSSION THREAD

**Discussion by santosh1b** [learndevopsonline/learndevopsonline#857] [Q&A]
hello ,,sir what is the full form of the SSH (key pair).
SSO is the single sign on  right?

**Comment by Arafique458** [comment]
SSH (Secure Shell), is a network protocol that allows you to securely connect to a remote server or computer. On May 14, 2023, at 9:38 PM, santosh1b ***@***.***> wrote:﻿
hello ,,sir what is the full form of the SSH (key pair).
SSO is the single sign on  right?
